src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtSpoeAreaOfLawEndpointTest.java

```diff
@@ -0,0 +1,195 @@
+            ABERYSTWYTH_JUSTICE_CENTRE_SPOE_AREAS_OF_LAW_PATH,
+            Map.of(AUTHORIZATION, BEARER + superAdminToken),
+            updatedJson
+        );
+        assertThat(response.statusCode()).isEqualTo(OK.value());
+
+        final List<AreaOfLaw> updatedCourtAreaOfLaw = response.body().jsonPath().getList(
+            ".",
+            AreaOfLaw.class
+        );
+        assertThat(updatedCourtAreaOfLaw.get(0).getName()).isEqualTo(expectedCourtAreasOfLaw.get(0).getName());
+        assertThat(updatedCourtAreaOfLaw.get(0).getId()).isEqualTo(expectedCourtAreasOfLaw.get(0).getId());
+
+        //clean up by removing added record
+        final var cleanUpResponse = doPutRequest(
+            ABERYSTWYTH_JUSTICE_CENTRE_SPOE_AREAS_OF_LAW_PATH,
+            Map.of(AUTHORIZATION, BEARER + superAdminToken),
+            originalJson
+        );
+        assertThat(cleanUpResponse.statusCode()).isEqualTo(OK.value());
+
+        final List<AreaOfLaw> cleanCourtAreaOfLaw = cleanUpResponse.body().jsonPath().getList(
+            ".",
+            AreaOfLaw.class
+        );
+        assertThat(cleanCourtAreaOfLaw.get(0).getName()).isEqualTo(currentCourtSpoeAreaOfLaw.get(0).getName());
+        assertThat(cleanCourtAreaOfLaw.get(0).getId()).isEqualTo(currentCourtSpoeAreaOfLaw.get(0).getId());
+
+    }
+
+    @Test
+    public void shouldRequireATokenWhenUpdatingSpoeAreaOfLawForTheCourt() throws JsonProcessingException {
+        final List<AreaOfLaw> currentCourtSpoeAreaOfLaw = getCurrentSpoeAreaOflaw();
+        final String testJson = objectMapper().writeValueAsString(currentCourtSpoeAreaOfLaw);
+        final var response = doPutRequest(ABERYSTWYTH_JUSTICE_CENTRE_SPOE_AREAS_OF_LAW_PATH, testJson);
+        assertThat(response.statusCode()).isEqualTo(UNAUTHORIZED.value());
+    }
+
+    @Test
+    public void shouldBeForbiddenForUpdatingSpoeAreaOfLawForTheCourt() throws JsonProcessingException {
+        final List<AreaOfLaw> currentCourtSpoeAreaOfLaw = getCurrentSpoeAreaOflaw();
+        final String testJson = objectMapper().writeValueAsString(currentCourtSpoeAreaOfLaw);
+        final var response = doPutRequest(
+            ABERYSTWYTH_JUSTICE_CENTRE_SPOE_AREAS_OF_LAW_PATH,
+            Map.of(AUTHORIZATION, BEARER + forbiddenToken), testJson
+        );
+        assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
+    }
+
+    @Test
+    public void shouldnotUpdateAndReturnNotFoundWhenSpoeCourtDoesNotExist() throws JsonProcessingException {
+        final List<AreaOfLaw> currentCourtSpoeAreaOfLaw = getCurrentSpoeAreaOflaw();
+        final String testJson = objectMapper().writeValueAsString(currentCourtSpoeAreaOfLaw);
+        final Response response = doPutRequest(
+            ABERYSTWYTH_COURT_AREAS_OF_LAW_NOT_FOUND_PATH,
+            Map.of(AUTHORIZATION, BEARER + superAdminToken),
+            testJson
+        );
+        assertThat(response.statusCode()).isEqualTo(NOT_FOUND.value());
+    }
+
+
```

Comment:
Should we add another test for checking for a conflict? Or is this better off being a manual/unit test?

Reply:
We agreed to keep it a manual test

Reply:
yes, Its manually tested.